src/sentry/static/sentry/app/views/usageStats/usageStatsProjects.tsx

```diff
@@ -169,14 +169,16 @@ class UsageStatsProjects extends AsyncComponent<Props, State> {
     });
   }
 
-  get tableLink() {
+  getTableLink(project: Project) {
     const {dataCategory, organization} = this.props;
 
     if (dataCategory === DataCategory.TRANSACTIONS) {
-      return `/organizations/${organization.slug}/performance/`;
+      return `/organizations/${organization.slug}/performance/?project=${project.id}`;
     }
 
-    return `/organizations/${organization.slug}/issues/`;
+    return organization.features.includes('project-detail')
+      ? `/organizations/${organization.slug}/projects/${project.slug}/?project=${project.id}`
+      : `/organizations/${organization.slug}/issues/?project=${project.id}`;
```

Comment:
Do you need to preserve the current time range?

Reply:
Fixed in https://github.com/getsentry/sentry/pull/25211/commits/4c2171fe2f0f5fd8f9785b7cf300db966d3f63ff